core/designsystem/src/main/java/com/susu/core/designsystem/component/badge/SusuBadge.kt

```diff
@@ -0,0 +1,132 @@
+package com.susu.core.designsystem.component.badge
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.susu.core.designsystem.theme.SusuTheme
+
+@Composable
+fun SusuBadge(
+    modifier: Modifier = Modifier,
+    shape: Shape = RoundedCornerShape(4.dp),
+    color: BadgeColor,
+    text: String,
+    padding: @Composable () -> BadgePadding,
+) {
+    val horizontalPadding = padding().horizontalPadding
+    val verticalPadding = padding().verticalPadding
+
+    Box(
+        modifier = modifier.background(color.backgroundColor, shape),
+    ) {
+        Text(
+            text = text,
+            style = SusuTheme.typography.title_xxxs,
+            color = color.textColor,
+            modifier = modifier.padding(
+                horizontal = horizontalPadding,
+                vertical = verticalPadding,
+            ),
+        )
+    }
```

Comment:
이렇게 하면 padding() 람다가 2번 실행되는 문제가 있습니다!

```suggestion
    with(padding()) {
    Box(
        modifier = modifier.background(color.backgroundColor, shape),
    ) {
        Text(
            text = text,
            style = SusuTheme.typography.title_xxxs,
            color = color.textColor,
            modifier = modifier.padding(
                horizontal = horizontalPadding,
                vertical = verticalPadding,
            ),
        )
    }
    }
```